draft-ietf-quic-manageability.md

```diff
@@ -1023,6 +1023,38 @@ not arrive at their destination. Admitting a few packets allows the QUIC
 endpoint to determine that the path accepts QUIC. Sudden drops afterwards will
 result in slow and costly timeouts before abandoning the connection.
 
+## Guiding Path MTU
+
+Some networks are required to be able to carry 1500-byte packets but become
+inefficient at this size, because such packets are fragmented and
+reassembled across a segment where the effective MTU is smaller (e.g. due to
+IPSEC overhead). They do this even when the encapsulated packet is not a
+candidate for fragmentation by routers (it's IPv6, or has DF bit set). In such
+networks this leads to a need to influence a majority of senders to use smaller
+packets, so that the limited reassembly capacity is not exceeded.
+
```

Comment:
This makes it sound like the current method is very wrong - It is **OK* for a lower layer (including a tunnel) to fragment and reassemble to the MTU it supports. However, in some cases it **CAN** fragmentation and reassembly below IP can add to the processing cost and memory requirements - and hence can become a bottleneck. This text isn't clear yet  - but the text might be heading in the correct direction

Reply:
Would it be better to say this?

"Some networks are required to support 1500-byte packets but can only do so by fragmenting at a lower layer before traversing a smaller MTU segment, and then reassembling. This is permissible even when the IP layer is IPv6 or has DF bit set, because it occurs at that lower layer. However this process can add to compute and memory costs, leading to a bottleneck that limits network capacity. In such networks this generates a desire to influence a majority of senders to use smaller packets, so that the limited reassembly capacity is not exceeded."

Reply:
Great - now applied.